tardis/montecarlo/src/cmontecarlo.c

```diff
@@ -139,12 +139,55 @@ binary_search (const double *x, double x_insert, int64_t imin,
   return ret_val;
 }
 
+void
+do_angle_aberration (rpacket_t *packet, const storage_model_t *storage)
```

Comment:
Since this is only active during `full_relativity` I'd do the relativity check in this function instead of outside.
That way you can't execute the function by accident and the code would be cleaner.

The disadvantage is you'd have to look at the function implementation to know this is `full_relativity` only. But I think we can live with that.